### PR TITLE
Create block-info docker

### PR DESCRIPTION
**Pull request by rberg2**

### CODE REVIEW

File: docker/sawtooth-int-block-info-tp [hunk 1]
```diff
@@ -21,7 +21,7 @@
 #   This image should be built using `build_all installed`.
 #
 # Run:
-#   $ docker run sawtooth-block-info-tp
+#   $ docker run sawtooth-int-block-info-tp
```
Comment (aludvik) [inline docker/sawtooth-int-block-info-tp]: This is not a typo, the Dockerfile is `sawtooth-int-block-info-tp` but the image created by build_all is called `sawtooth-block-info-tp`.

**Comment by rberg2** [comment]
@aludvik I have dropped the commit to docker/sawtooth-int-block-info-tp

File: ci/sawtooth-validator [hunk 2]
```diff
@@ -33,6 +33,7 @@ RUN echo "deb http://repo.sawtooth.me/ubuntu/1.0/stable xenial universe" >> /etc
  && apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 8AA7AF1F1091A5FD \
  && apt-get update \
  && apt-get install -y -q \
+    python3-sawtooth-block-info \
```
Comment (aludvik) [inline ci/sawtooth-validator]: This should be in a separate commit since it isn't related to the original commit message.

Comment (rberg2) [reply ci/sawtooth-validator]: I moved this to a separate commit as requested.
